src/ZoneServer/Pads/Handlers/Swordsman/Hoplite/ThrowingSpear_Hoplite33.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Threading.Tasks;
+using Melia.Shared.Game.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Melia.Zone.World.Actors.Monsters;
+using Melia.Zone.World.Actors.Pads;
+using Yggdrasil.Logging;
+
+namespace Melia.Zone.Pads.Handlers.Swordsman.Hoplite
+{
+	/// <summary>
+	/// Handler for the ThrouwingSpear_Hoplite33_Pad,
+	/// which displays an explosion effect in a given position
+	/// </summary>
+	[PadHandler("ThrouwingSpear_Hoplite33_Pad")]
```

Comment:
You should consider using the static `PadName` class, as it may prevent accidental typos.

Reply:
I expected it would be that, but I actually based it off your implementation for the Shield Lob pad, which is specified using a string.

Reply:
Shame on me, forgot to update Shield Lob^^